mods/ts/rules/structures.yaml

```diff
@@ -127,6 +127,31 @@ GAPOWRUP:
 	Power:
 		Amount: 50
 
+GASPOT:
```

Comment:
By that, @pchote meant it is not (and was not intended to be) built by the players.